jboss-eap-cd16-openshift/added/standalone-openshift.xml

```diff
@@ -480,6 +481,10 @@
             <!-- ##MICROPROFILE_CONFIG_SOURCE## -->
         </subsystem>
         <subsystem xmlns="urn:wildfly:microprofile-health-smallrye:1.0"/>
+        <subsystem xmlns="urn:wildfly:microprofile-metrics-smallrye:2.0"
+            security-enabled="false"
+            exposed-subsystems="*"
+            prefix="${wildfly.metrics.prefix:jboss}"/>
```

Comment:
Should it be jboss-eap instead of jboss ?

Reply:
Metrics that are exposed by EAP (using Prometheus JMX agent) already use the `jboss` prefix. https://github.com/jboss-container-images/jboss-eap-modules/blob/master/jboss/container/eap/prometheus/config/7.2/artifacts/opt/jboss/container/prometheus/etc/jmx-exporter-config.yaml#L43

Reply:
That something is already done does not mean it is correct

Reply:
there is no product jboss only jboss-eap